pages/index.tsx

```diff
@@ -29,93 +29,99 @@ import { StickyHeader } from '../components/sticky-header/sticky-header';
 import MultiplatformPreviewImage from '../public/images/main/multiplatform-preview.svg';
 
 import GradleLogo from '../public/images/companies/gradle.svg';
-import CordaLogo from '../public/images/companies/corda.svg';
-import EvernoteLogo from '../public/images/companies/evernote.svg';
-import CourseraLogo from '../public/images/companies/coursera.svg';
-import SpringLogo from '../public/images/companies/spring.svg';
 import AtlassianLogo from '../public/images/companies/atlassian.svg';
 import GoogleLogo from '../public/images/companies/google.svg';
 import JetbrainsLogo from '../public/images/companies/jetbrains.svg';
 import ShopifyLogo from '../public/images/companies/shopify.svg';
 import TouchlabLogo from '../public/images/companies/touchlab.svg';
 
+import McDonaldsLogo from '../public/images/companies/mcdonalds.svg';
+import AWSLogo from '../public/images/companies/aws.svg';
+import PhilipsLogo from '../public/images/companies/philips.svg';
+import AdobeLogo from '../public/images/companies/adobe.svg';
+import ForbesLogo from '../public/images/companies/forbes.svg';
+
 import styles from './index.module.css';
 
 import searchConfig from '../search-config.json';
 
 
-
 import releasesDataRaw from '../data/releases.yml';
 
 const releasesData: ReleasesData = releasesDataRaw as ReleasesData;
 
 
-
 const kotlinUsageHighlightsCases = [
     {
-        company: 'Gradle',
-        url: 'https://blog.gradle.org/kotlin-meets-gradle',
-        text: 'Gradle is introducing Kotlin as a language for writing build scripts',
-        logo: GradleLogo,
+        company: 'McDonald\'s',
+        url: 'https://medium.com/mcdonalds-technical-blog/mobile-multiplatform-development-at-mcdonalds-3b72c8d44ebc',
+        text: 'McDonald\'s leverages Kotlin Multiplatform (KMP) for their global mobile app, enabling them to build a codebase that can be shared across platforms, removing the need for codebase redundancies.',
+        tag: 'Kotlin Multiplatform',
+        logo: McDonaldsLogo
     },
     {
-        company: 'Corda',
-        url: 'https://www.corda.net/2017/01/10/kotlin/',
-        text: 'Corda is an open-source distributed ledger platform, supported by major banks, and built entirely in Kotlin',
-        logo: CordaLogo,
+        company: 'AWS',
+        url: 'https://talkingkotlin.com/qldb/?_ga=2.262984399.2130827358.1713171937-1458617801.1712600110&_gac=1.82911588.1710842740.CjwKCAjw7-SvBhB6EiwAwYdCAc1D_e4EpUSLWofp6yNeKb9uLurBFqJ4szLvydLezOUS-l9WUOJEzhoCEoIQAvD_BwE&_gl=1*hyh7xv*_ga*MTQ1ODYxNzgwMS4xNzEyNjAwMTEw*_ga_9J976DJZ68*MTcxMzM3MDQ1MC4yMzMuMS4xNzEzMzcwOTkyLjUzLjAuMA..',
```

Comment:
I doubt the link was intentionally shared with us with the params. Let's remove it.
```suggestion
        url: 'https://talkingkotlin.com/qldb/',
```